packages/grid/docker-compose.multinode.yml

```diff
@@ -0,0 +1,461 @@
+      - VERSION_HASH=${VERSION_HASH}
+      - NODE_TYPE=${NODE_TYPE?Variable not set}
+      - NODE_NAME=${NODE_NAME?Variable not set}
+      - STACK_API_KEY=${STACK_API_KEY}
+      - IGNORE_TLS_ERRORS=${IGNORE_TLS_ERRORS?False}
+      - USE_BLOB_STORAGE=${USE_BLOB_STORAGE}
+      - CONTAINER_HOST=${CONTAINER_HOST}
+      - TRACE=${TRACE}
+      - JAEGER_HOST=${JAEGER_HOST}
+      - JAEGER_PORT=${JAEGER_PORT}
+      - ASSOCIATION_TIMEOUT=${ASSOCIATION_TIMEOUT}
+      - DOMAIN_CONNECTION_PORT=${DOMAIN_CONNECTION_PORT}
+      - ENABLE_OBLV=${ENABLE_OBLV}
+      - DEFAULT_ROOT_PASSWORD=${DEFAULT_ROOT_PASSWORD}
+      - BACKEND_STORAGE_PATH=${BACKEND_STORAGE_PATH}
+      - QUEUE_PORT=${QUEUE_PORT}
+      - CREATE_PRODUCER=false
+      - N_CONSUMERS=1
+      - DEFAULT_ROOT_USERNAME=worker1
+      - PORT=8002
+      - HTTP_PORT=87
+      - HTTPS_PORT=445
+      - DEFAULT_ROOT_EMAIL=info2@openmined.org
+      - DEV_MODE=True
+    command: "/app/grid/start.sh"
+    network_mode: service:proxy
+    volumes:
+      - credentials-data-worker:/storage
+      - ${RELATIVE_PATH}./backend/grid:/app/grid
+      - ${RELATIVE_PATH}../syft:/app/syft
+      - ${RELATIVE_PATH}./data/package-cache:/root/.cache
+    stdin_open: true
+    tty: true
+    labels:
+      - "orgs.openmined.syft=this is a syft backend container"
+
+  backendworker2:
+    restart: always
+    image: "${DOCKER_IMAGE_BACKEND?Variable not set}:${VERSION-latest}"
+    depends_on:
+      - proxy
+      - backend
+    env_file:
+      - .env
+    environment:
+      - SERVICE_NAME=backend
+      - RELEASE=${RELEASE:-production}
+      - VERSION=${VERSION}
+      - VERSION_HASH=${VERSION_HASH}
+      - NODE_TYPE=${NODE_TYPE?Variable not set}
+      - NODE_NAME=${NODE_NAME?Variable not set}
+      - STACK_API_KEY=${STACK_API_KEY}
+      - IGNORE_TLS_ERRORS=${IGNORE_TLS_ERRORS?False}
+      - USE_BLOB_STORAGE=${USE_BLOB_STORAGE}
+      - CONTAINER_HOST=${CONTAINER_HOST}
+      - TRACE=${TRACE}
+      - JAEGER_HOST=${JAEGER_HOST}
+      - JAEGER_PORT=${JAEGER_PORT}
+      - ASSOCIATION_TIMEOUT=${ASSOCIATION_TIMEOUT}
+      - DOMAIN_CONNECTION_PORT=${DOMAIN_CONNECTION_PORT}
+      - ENABLE_OBLV=${ENABLE_OBLV}
+      - DEFAULT_ROOT_PASSWORD=${DEFAULT_ROOT_PASSWORD}
+      - BACKEND_STORAGE_PATH=${BACKEND_STORAGE_PATH}
+      - QUEUE_PORT=${QUEUE_PORT}
+      - CREATE_PRODUCER=false
+      - N_CONSUMERS=1
+      - DEV_MODE=True
+      - DEFAULT_ROOT_USERNAME=worker2
+      - PORT=8003
+      - HTTP_PORT=88
+      - HTTPS_PORT=446
+      - DEFAULT_ROOT_EMAIL=info3@openmined.org
+    command: "/app/grid/start.sh"
+    network_mode: service:proxy
+    volumes:
+      - credentials-data-worker2:/storage
+      - ${RELATIVE_PATH}./backend/grid:/app/grid
+      - ${RELATIVE_PATH}../syft:/app/syft
+      - ${RELATIVE_PATH}./data/package-cache:/root/.cache
+    stdin_open: true
+    tty: true
+    labels:
+      - "orgs.openmined.syft=this is a syft backend container"
+
+  backendworker3:
```

Comment:
We might want to use replicas over here instead of different instances of backendworkers

Reply:
see other comment